WeatherInfo/ViewControllers/SettingsViewController/SettingsViewController.swift

```diff
@@ -117,6 +125,11 @@ extension SettingsViewController: UITableViewDataSource, UITableViewDelegate {
         return cell
     }
 
+    func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        guard let section = Section(rawValue: section) else { fatalError("Unexpected Section") }
```

Comment:
Why you want fatalError. The fatal error is crash. You dont want to introduce crash yourself. If something is unexpected you would want some graceful exit

Reply:
Right, need to improve this. Maybe when no section title is fetched it can remain empty, will not impact on Apps performance or experience.

Reply:
Going with the above approach, keeping header empty instead of crashing.